*: update guiding materials in docs-cn repo (#2788)
pingcap/docs-cn: .github/pull_request_template.md

### Which TiDB version(s) do your changes apply to? (Required)

> <!-- Fill in "x" in [] (i.e. [x]) or tick the checkbox(es) below to choose the TiDB version(s) that your changes apply to.-->

Author: ```suggestion
<!-- Fill in "x" in [] (that is, [x]) or tick the checkbox(es) below to choose the TiDB version(s) that your changes apply to.-->
```

- [ ] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

> <!-- For contributors with write access in docs-cn repo:

Reviewer: ```suggestion
<!-- For contributors with write access in the docs-cn repo:
```
